consumer/dragonboard/dragonboard410c/installation/board-recovery.md

```diff
@@ -2,8 +2,7 @@
 title: DragonBoard 410c Board Recovery
 permalink: /documentation/consumer/dragonboard/dragonboard410c/installation/board-recovery.md.html
 redirect_from:
-- /db410c-getting-started/
```

Comment:
Why did db410c-getting-started/ redirect get removed? I've just checked the routingrules.json file in the website repo and it's not listed there either :)

Reply:
You can only have a URL mentioned once under "redirect_from" otherwise you end up with multiple pages all trying to be the destination for the redirect page.

db410c-getting-started still exists in consumer > dragonboard > dragonboard410c > README.md which seemed to be the best place.